src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -470,6 +470,73 @@ The main troubles you may have to face are the following ones:
 * You get an *authentication error*: this is probably due to your default credentials file which cannot be found. In the "Connection" tab of the Configuration Editor (Figure 5.4, “Configuration Editor window - Connection Tab (Resource Manager Registration)”), you can choose the credentials file you want. You can select, for instance, the credentials file located at PROACTIVE_HOME/config/authentication/scheduler.cred or your own credentials file.
 * The node seems to be well started but you cannot see it in the Resource Manager interface : in this case, make sure that the *port number* is the good one. Do not forget that the runtime port number is incremented from the initial ProActive Resource Manager port number. You can see exactly on which port your runtime has been started looking at the log file described above.
 
+==== Install Agent Windows Using Silent Mode
+
+When installing the agent in batch on several desktop machines one might use the
+command line installer instead of the graphical one. This eases to deploy several
+installations in parallel. To do so it is requiered to have remote access to
+command prompt with full admin priviledges.
+
+The silent option works to execute the installer directly on the command line
+without the need for a graphical interfaces. When the silent option is activated
+we need to provide at least user and password as parameters, domain name is optional
+since it is automatically detected.
+
+
+For instance to install the latest windows agent go
+link:http://jenkins.activeeon.com/job/agent-windows/lastSuccessfulBuild/artifact/[here]
```

Comment:
I don't think our jenkins platform is to be used publicly. Instead, we should give the link to either the respective git repo (for the community edition) or the breadcrumb navigation link through the client's account page on the activeeon website (where we usually provide those builds). What do you think @fviale  ?